### PR TITLE
Now

### PR DESCRIPTION
**Pull request by FintanH**

### CODE REVIEW

**Comment by bitemyapp** [comment]
@FintanH https://travis-ci.org/bitemyapp/esqueleto/builds/260756675?utm_source=github_status&utm_medium=notification couple things to fix in the build, one of them dependency lower bound, another the imports.

Even after you fix those, I think for the tests to be correct and passing you'd want to wrap this test stanza for `now_` in the PostgreSQL-only cpp macro right?

File: esqueleto.cabal [hunk 1]
```diff
@@ -102,7 +102,7 @@ test-suite test
     , persistent-template >= 2.1
     , monad-control
     , monad-logger        >= 0.3
-    , time                >= 1.6.0.1
+    , time                >= 1.5.0.1 && <= 1.6.0.1
```
Comment (bitemyapp) [inline esqueleto.cabal]: higher upper bound than that please, need to test it with 1.8.0.1

Comment (FintanH) [inline esqueleto.cabal]: Ah ok will do 👍

**Comment by bitemyapp** [comment]
Thank you @FintanH!

**Pull request merged**